stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -739,9 +745,13 @@ internal class ChatDomainImpl internal constructor(
         return retriedMessages
     }
 
-    private suspend fun retryMessagesWithoutAttachments(): List<Message> {
+    private suspend fun retryMessagesWithSyncedAttachments(): List<Message> {
```

Comment:
This still includes messages that have no attachments, right? Maybe it's worth documenting that either in the method name (which will get quite long, but it's be more descriptive), or with actual comments.